dts/arm/st/mem.h

```diff
@@ -11,6 +11,12 @@
 #if defined(CONFIG_SOC_STM32F030X8)
 #define DT_FLASH_SIZE		__SIZE_K(64)
 #define DT_SRAM_SIZE		__SIZE_K(8)
+#elif defined(CONFIG_SOC_STM32F091XB)
```

Comment:
While this is correct, I wonder why STM32F091XB is added here.
Please remove as 091XB is not supported otherwise

Reply:
Done.